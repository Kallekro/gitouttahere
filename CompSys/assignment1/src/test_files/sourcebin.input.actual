test_files/sourcebin.input: data
