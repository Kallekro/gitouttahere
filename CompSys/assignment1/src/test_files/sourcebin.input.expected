test_files/sourcebin.input: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=9d996c56a6a94dd217b904db946b14984f85c9cf, not stripped
